brave/utility/importer/brave_profile_import_service.cc

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2017 The Brave Authors. All rights reserved.
```

Comment:
I'm confused, this looks identical to `chrome/utility/importer/profile_import_service.cc`. What am I missing?

Reply:
we need `BraveProfileImportImpl` which is the only difference than `chrome/utility/importer/profile_import_service.cc`